server/services/prisonerCellAllocationService.ts

```diff
@@ -18,11 +18,6 @@ export default class PrisonerCellAllocationService {
     return await this.prisonApiClient.getInmates(token, locationId, keywords, returnAlerts)
   }
 
-  // Deprecated TODO: Remove this code and update all controllers to use getPrisonersAtLocations instead
```

Comment:
replaced by locationsInsidePrisonApiClient.getInmatesAtLocation

Reply:
How often is this called for a page?
i.e. is it a n+1 issue